ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -176,7 +179,9 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
       })
       .subscribe((nbName) => {
         this.progress = Progress.Redirecting;
-        this.window.location.href = this.notebookUrl(this.cluster, nbName);
+        this.notebookLoaded = true;
```

Comment:
Note: It would be nicer if we waited for some loading status from the iframe before showing it. Not for this PR but possible future improvement.